### PR TITLE
Forgot password test cases

### PR DESCRIPTION
**Pull request by ninanichik**
In file oct/tests/web/forgot_password_test.py was added new negative test case for forgot password functional

### CODE REVIEW

File: oct/pages/forgot_password.py [hunk 1]
```diff
@@ -35,3 +35,10 @@ def loaded(self) -> bool:
             return True
         except NoSuchElementException:
             return False
+
+    def not_loaded(self) -> bool:
```
Comment (extsoft) [inline oct/pages/forgot_password.py]: @ninanichik Why do we need this method? We can use `not page.loaded()`.

File: oct/tests/web/forgot_password_test.py [hunk 2]
```diff
@@ -17,5 +17,16 @@ def test_forgot_password(self, grid: str) -> None:
         assert ConfirmationMessage(chrome).loaded()
 
 
+class ForgotPasswordNegativeTest(Testcase):
```
Comment (AlekseyDidilov) [inline oct/tests/web/forgot_password_test.py]: @ninanichik Why do you create new "ForgotPasswordNegativeTest" class, maybe better move method "test_forgot_password_with_not_existing_email" in "ForgotPassword"?

**Comment by extsoft** [comment]
@ninanichik please squash the commits.

File: oct/pages/forgot_password.py (hunk 1 above)
Comment (ninanichik) [reply oct/pages/forgot_password.py]: Fixed

File: oct/tests/web/forgot_password_test.py (hunk 2 above)
Comment (ninanichik) [reply oct/tests/web/forgot_password_test.py]: @AlekseyDidilov  fixed

**Comment by ninanichik** [comment]
> @ninanichik please squash the commits.

Thanks! I have already done this

**Pull request merged**